portal-impl/src/portal.properties

```diff
@@ -6644,6 +6644,16 @@
     #
     feature.flag.LPS-189856=false
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER9__NUMBER2__NUMBER0__NUMBER5__NUMBER3_
+    #
+    feature.flag.LPS-192053=false
```

Comment:
I think we don't need this feature flag, as we are using the one for LPS-192957, am I right?